spec/features/settings/dex_connector_ldap_feature_spec.rb

```diff
@@ -130,6 +130,75 @@
     end
   end
 
+  # rubocop:disable RSpec/ExpectInHook
+  describe "#new_save_disabled_by_change" do
+    before do
+      visit new_settings_dex_connector_ldap_path
+      expect(page.evaluate_script("$('#ldap_conn_save').prop('disabled')")).to eq(true)
+
+      # Test that we can enable the button
+      page.execute_script("$('#ldap_conn_save').removeProp('disabled')")
+      expect(page.evaluate_script("$('#ldap_conn_save').prop('disabled')")).to eq(false)
+    end
+
+    it "allows change of start_tls" do
+      # Is false by default - change to true
+      # Following line should work but it does not so using JS
+      # choose id: "dex_connector_ldap_start_tls_true"
+      page.execute_script("$('#dex_connector_ldap_start_tls_true').prop('checked', true)")
+      page.execute_script("$('#dex_connector_ldap_start_tls_true').trigger('change')")
+    end
+
+    it "allows change of bind_anon" do
+      # Is false by default - change to true
+      # Following line should work but it does not so using JS
+      # choose id: "dex_connector_ldap_bind_anon_true"
+      page.execute_script("$('#dex_connector_ldap_bind_anon_true').prop('checked', true)")
```

Comment:
You can add an id/class to the label element and use `find` and `click` to mimic the user behavior instead of changing the value directly through `execute_script`. By doing that you are bypassing the normal behavior of the engine that might lead into unexpected results like not triggering an event.